cms/envs/aws.py

```diff
@@ -265,6 +267,11 @@
     DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
 
 DATABASES = AUTH_TOKENS['DATABASES']
+
+# Enable automatic transaction management on all databases
+for database_name in DATABASES:
+    DATABASES[database_name]['ATOMIC_REQUESTS'] = True
```

Comment:
@feanil We will always need this to be true which is why it is being set like this. I'll add a note about its purpose and a link to the Django docs: https://docs.djangoproject.com/en/1.8/topics/db/transactions/#tying-transactions-to-http-requests